src/views/components/input/file/InputFile.web.js

```diff
@@ -232,13 +215,13 @@ class InputFile extends Component {
       debug: false,
       restrictions: {
         maxNumberOfFiles: this.props.maxNumberOfFiles,
-        allowedFileTypes: this.props.allowedFileTypes,
+        // allowedFileTypes: this.props.allowedFileTypes,
       },
       onBeforeFileAdded: ( currentFile ) => this.checkFileType( currentFile ),
     })
       .use( Dashboard, {
         closeModalOnClickOutside: true,
-        note: '.jpeg, .jpg, and .png file types allowed only',
+        note: this.props.imageOnly ? '.jpeg, .jpg, and .png file types allowed only' : 'any file type allowed',
```

Comment:
Can we please expand this ternary condition to 3 lines to make it easier to read?